src/makeRobot.js

```diff
@@ -38,7 +38,78 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    version: version,
+    wheels: wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, wheels: `
+      + this.wheels;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value) {
+      if (value === undefined) {
+        this.coords.y++;
+      }
```

Comment:
You don't need to check if a value is undefined if you'll add a default value. Fix it everywhere, please
```suggestion
    goForward(value = 1) {
```